From ec57094798856264e7c37d93b92a81cf22e0b491 Mon Sep 17 00:00:00 2001
From: Saurabh Shah <saurshah@codeaurora.org>
Date: Tue, 7 Jan 2014 17:29:03 +0000
Subject: [PATCH 2/6] qdutils: Use HW revision from linux header if available

Currently HW revisions are maintained separately between kernel and
user-space. To optimize this there will be single place, exported
msm_mdp.h, where HW revisions will be maintained.

Change-Id: I815eb9bd32f17928356e58a38260cee69b019a8b

Partial backport from 693d5c2fa030ce90b0f8ccd5cffd255525e2305c.
Keep the CM11 revision enum in its existing header and guard it with MDSS_MDP_REV; do not import the newer revision-query API.
---
 libqdutils/mdp_version.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/libqdutils/mdp_version.h b/libqdutils/mdp_version.h
index eb33d3f..cacbe79 100644
--- a/libqdutils/mdp_version.h
+++ b/libqdutils/mdp_version.h
@@ -52,11 +52,13 @@ enum mdp_version {
     MDSS_V5     = 500,
 };
 
+#ifndef MDSS_MDP_REV
 enum mdp_rev {
     MDSS_MDP_HW_REV_100 = 0x10000000,
     MDSS_MDP_HW_REV_101 = 0x10010000, //8x26
     MDSS_MDP_HW_REV_102 = 0x10020000,
 };
+#endif
 
 #define MDDI_PANEL       '1'
 #define EBI2_PANEL       '2'
-- 
2.53.0

